plugins/techdocs/src/api.ts

```diff
@@ -152,9 +166,14 @@ export class TechDocsStorageApi implements TechDocsStorage {
 
     const apiOrigin = await this.getApiOrigin();
     const url = `${apiOrigin}/docs/${namespace}/${kind}/${name}/${path}`;
+    const token = await this.identityApi.getIdToken();
 
     const request = await fetch(
       `${url.endsWith('/') ? url : `${url}/`}index.html`,
+      {
+        credentials: 'include',
```

Comment:
I didn't get the cookie to be set without it, despite Set-Cookie header being set from the backend. My googling seemed to suggest that for client-side requests it is necessary or the `Set-Cookie` header is ignored. I agree that it is an odd bird and as such nice to remove. Ideas?

Reply:
Ah hmm, so at some point we need a `fetch` request that includes credentials so that it can set the cookie for all non-xhr requests? Prolly not great to have that happening within the TechDocs plugin either tbh, perhaps the best home for that is the sign-in page? I think you might even be able to set up a timer there to refresh the cookie when needed.

Reply:
To me, it makes sense to match the cookie with the token, e.g. set the cookie on successful login and refresh the cookie on successful token refresh. We did want to give that discussion some more time/space though. I guess what we might want to do is move this bit too into adopter-country, e.g. something I can add on my end without polluting core. That should be doable, and on client-side (http only is not so important since we already have the token in localStorage). I will remove this bit.

Reply:
Alright, we don't have tokens in `localStorage` though, with the exception of GitHub because GitHub is weird. Other tokens are "stored" as refresh tokens in http-only secure cookies

Reply:
I see. Well, a new custom route in the api backend then to set the cookie, and some client-side calling of same at regular intervals.